migrations/versions/0348_migrate_broadcast_settings_migrate_broadcast_settings.py

```diff
@@ -0,0 +1,48 @@
+"""
+
+Revision ID: 0348_migrate_broadcast_settings
+Revises: 0347_add_dvla_volumes_template
+Create Date: 2021-02-18 15:25:30.667098
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0348_migrate_broadcast_settings'
+down_revision = '0347_add_dvla_volumes_template'
+
+
+def upgrade():
+  # For every service that has the broadcast permission we want it to have
+  # a row in the broadcast_service_settings table
+  # 
+  # If it doesnt have a row already, then:
+  # - if the service is in trial mode, add a row and set the channel as 'severe'
```

Comment:
Just to check my understanding: for the current test channel services in trial mode, does this mean we'll need to manually amend them in each environment? If so, do we have a good idea which services these are?

Reply:
I don't think we will need to do any manual intervention, this migration should take care of all services and put them in a legit state (one that meets one of the radio options on the platform admin page to choose a broadcast account type).

For current broadcast services in trial mode, they will be be changed by this migration to be in trial mode but instead send to the severe channel (rather than default to the test channel). This change shouldn't really have an impact on any services. On preview there will be a bunch of services that fall into this category but that is OK as it's our preview environment so we don't really mind what channel it gets sent out on. For production, as training mode services don't send alerts, again we don't really mind if we change their channel from test to severe.

Let me know if you want me to chat this through in person if that is easier.